index.rst

```diff
@@ -379,52 +355,50 @@ In order to merge a branch to the develop branch, each job must:
       It does not check format/readability/sensible inputs etc.
 
 
-No integration testing (on the ncsa-teststand) is strictly required, however, one would hope that the script has run
+No integration testing (on the NCSA-teststand) is strictly required, however, one would hope that the script has run
 successfully through the integration-test-stand or on the summit.
 
 
-Jobs in Production
-^^^^^^^^^^^^^^^^^^
+Scripts in Production
+^^^^^^^^^^^^^^^^^^^^^
 
-Jobs in full production are to be kept in the `ts_queueJobs` repository. This is the last step in the
-development process. Jobs in this category are tightly controlled and standards are strictly enforced. No production
-level Job can call any utility in the `Observing Utilities`_ repository. All utilities must be sanctioned Control
-Package Utilities. All jobs in this repository must be runnable at all times by any operator. All code
-requires documentation to a level where other developers can diagnose the
-code and fix any issues that are resulting in failed tests. This must include a description of the Job, a
+Scripts in full production are to be kept in the `ts_scripts` repository. This is the last step in the
+development process. Scripts in this category are tightly controlled and standards are strictly enforced. No production
+level script can call any utility in the `Observing Utilities`_ repository. All called utilities shall be sanctioned
```

Comment:
We could add that this will be enforced by the CI system. Something like,

> ... level script can call any utility in the Observing Utilities repository, which will be enforced during the CI process by making sure the library is not available.

Or maybe this is too much information? I don't know...

Reply:
I think we can add that once it exists :)